File: 2026/2248/2248a.cpp
/*!
 * author: manhhomienbienthuy
 * created: 2026-08-02T03:19:28+00:00
 * https://codeforces.com/contest/2248/problem/A
 */

#include <bits/stdc++.h>
using namespace std;

int main() {
  ios::sync_with_stdio(false);
  cin.tie(nullptr);

  int t;
  cin >> t;

  while (t--) {
    string s;
    cin >> s;

    auto p = s.find('0');
    s.erase(p, 1);

    p = s.find('1');
    s.erase(p, 1);

    cout << s << '\n';
  }

  return 0;
}

File: 2026/2248/2248a.py
# author: manhhomienbienthuy
# created: 2026-08-02T02:49:11+00:00
# https://codeforces.com/contest/2248/problem/A

import sys
input = sys.stdin.readline

t = int(input())
for _ in range(t):
    s = input().strip()
    s = s.replace('0', '', 1)
    s = s.replace('1', '', 1)
    print(s)
